lib/autoproj/system.rb

```diff
@@ -6,6 +6,14 @@ class UserError < RuntimeError; end
     def self.create_symlink(from, to)
         if Autobuild.windows?
             Dir.create_junction(to, from)
+        elsif Autobuild.msys?
+            begin
+                #msys created a copy instead of a link
+                FileUtils.ln_sf from, to
+            rescue Errno::EEXIST
+                FileUtils.rm_rf to
+                FileUtils.ln_sf from, to
+            end
```

Comment:
I don't really know what to do here, msys2 internally avoids symlinks and is doing a copy instead, so at the second execution, FileUtils.ln_sf throws Errno::EEXIST.

Ignoring the call leads to non-updating autoproj/remotes folders, deleting them works, but local changes could be lost